post/templates/post/add_comment.html

```diff
@@ -0,0 +1,25 @@
+{% extends 'post/base.html' %}
+{% block title %} New Comment {% endblock %}
+{% block content %}
+   
+ <h1>Add Comment</h1>
+ <div class="form-group">
+    <form method="POST">
+        {% csrf_token %}
+   {% if user.is_authenticated %}
+        {{ form.media }}
+   {% endif %}
+        {{ form.as_p }}
+        <br/>
+        <button class="btn btn-secondary"> Add Comment </button>
+    </form>
+</div>
+
+<script>
+    var name = "{{ user.username }}";
+    document.getElementById("users_id").value
+             = name;
```

Comment:
why is javascript used here?

Reply:
answered above

Reply:
This will show the currently logged user.

Reply:
removed jave script